nuts/base_tests/napalm_interfaces.py

```diff
@@ -35,21 +35,25 @@ def nuts_extractor(self) -> InterfacesExtractor:
 
 class TestNapalmInterfaces:
     @pytest.mark.nuts("name,is_enabled")
-    def test_is_enabled(self, single_result, name, is_enabled):
+    def test_is_enabled(
+        self, single_result: NutsResult, name: Any, is_enabled: Any
+    ) -> None:
```

Comment:
add more specific type

Reply:
Done, read the value types from the demo test bundle ".yaml"-files.